### ISSUE TITLE
Suggestion: supporting materials for aria-required-attr rule ID

### ISSUE THREAD

**Issue by dboudreau** [dequelabs/axe-core#216]
Based on comments received by a client, I suggest we provide a mapping or a comprehensive list of all required attributes for each aria roles that Axe checks for. That way, it would be easier for our users to understand what an issue is about when the "aria-required-attr" rule ID is triggered. To most, this currently feels like a black box.

**Comment by dylanb** [comment]
@dboudreau the documentation for this is the ARIA spec itself, all we have done is encode the requirements of the ARIA spec into the rules. If we duplicate that spec, we are creating a large document that we have to maintain for very little additional benefit

**Comment by dboudreau** [comment]
@dylanb Ah, yeah, I see how that could become a management nightmare over time. Ok, then I'll make it a 2do item on my end to come up with some kind of a supporting resource to give our clients instead.
